.github/workflows/format.yml

```diff
@@ -0,0 +1,28 @@
+name: Format (Check)
+
+on:
+  workflow_call:
+  # Make this a reusable workflow, no value needed
+  # https://docs.github.com/en/actions/using-workflows/reusing-workflows
+
+jobs:
+  format:
+    name: Check
+    runs-on: 'ubuntu-latest'
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v2.4.0
+
+      - name: Use Node 16
+        uses: actions/setup-node@56337c425554a6be30cdef71bf441f15be286854 # v3.1.1
+        with:
+          node-version: 16
+          cache: 'npm'
+
+      - name: Install Dependencies
+        run: npm ci
+        shell: bash
+
+      - name: Prettier Check
+        run: npm run prettier.dry-run
```

Comment:
Is this supposed to fail if prettier was not run?

Reply:
Yep! The `dry-run` command runs `prettier --list-different`, which gives a non-zero exit code if any non-formatted files are detected